generator/src/main/java/com/scottlogic/deg/generator/outputs/targets/OutputTarget.java

```diff
@@ -8,4 +8,7 @@
 
 public interface OutputTarget {
     void outputDataset(Stream<GeneratedObject> generatedObjects, ProfileFields profileFields) throws IOException;
+    boolean exists();
```

Comment:
This is all behaviour specific to a file output target, not a general output target.

Reply:
We need to raise an issue if we want to refactor the OutputTarget class hierarchy